dashboard/app/controllers/followers_controller.rb

```diff
@@ -60,7 +60,7 @@ def load_section
       return
     end
 
-    @section = Section.find_by_code(params[:section_code])
+    @section = Section.find_by_code(params[:section_code].strip)
```

Comment:
It would be great to add a test for this change so that we will prevent this issue from coming back up in the future.  There are a bunch of tests already for this file. I think you will want to write a test similar to https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/controllers/followers_controller_test.rb#L26 but that has a section code that would need to be stripped to work correctly. Let me know if you want to work through this together.  Here is the guide to running tests locally: https://github.com/code-dot-org/code-dot-org/blob/staging/TESTING.md#dashboard-tests.